update workbase docs to be compatible with latest workbase 1.2.0

Irtazaraza opened the pull request:
Update workbase images to match latest workbase off master

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

Commit:
Updates for new clients (#84)

- Fixes tests that failed because of the latest changes in clients nodejs and python.
- Updates the incomplete code snippets in migration and query tutorials.
- Update api references for clients nodejs and python.

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

Commit:
Remove client pypi dep (#87)

* define dir in imports of py_tests

* remove client pypi dep in requirements.txt

* update client-nodejs dep

kasper-piskorski committed:
Expand match clause section (#83)

* expand the match clause

* small corrections

* revise the extended match clause content

* ran grammarly

* small corrections

Commit:
Update workbase docs to be compliant with workbase 1.1.1 (#52)

Make workbase docs compliant with workbase 1.1.1

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

Commit:
update workbase docs to be compatible with latest workbase v1.1.2

Commit:
remove workbase version

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_nodejs dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_nodejs dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_nodejs dependency to latest 'master' branch

sorsaffari commented:
@Irtazaraza please pull `graknlabs/docs/development` into your `workbase-1.1.2` branch to get the tests to pass.

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_grakn_core dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_nodejs dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_nodejs dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_nodejs dependency to latest 'master' branch

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update @graknlabs_build_tools dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_java dependency to latest 'master' branch

grabl committed:
Update @graknlabs_client_python dependency to latest 'master' branch

grabl committed:
Update dependency to graknlabs/build-tools@7dd2298

grabl committed:
Update dependency to graknlabs/client-java@b548c05

grabl committed:
Update dependency to graknlabs/grakn@4970e45

Commit:
Merge branch 'development' of https://github.com/graknlabs/docs into workbase-1.1.2

grabl committed:
Update dependency to graknlabs/client-java@d456d79

grabl committed:
Update dependency to graknlabs/client-python@20b0b2e

grabl committed:
Update dependency to graknlabs/client-nodejs@67ed9eb

grabl committed:
Update dependency to graknlabs/client-nodejs@323d73f

grabl committed:
Update dependency to graknlabs/grakn@9133308

grabl committed:
Update dependency to graknlabs/client-java@a0ee6a5

grabl committed:
Update dependency to graknlabs/client-python@900474e

grabl committed:
Update dependency to graknlabs/client-nodejs@c5ed212

grabl committed:
Update dependency to graknlabs/grakn@540bc56

grabl committed:
Update dependency to graknlabs/client-java@8f7b276

grabl committed:
Update dependency to graknlabs/client-python@d2e02cb

grabl committed:
Update dependency to graknlabs/client-nodejs@e651854

vmax committed:
Installation docs for RPM/DEB/Docker (#90)

Installation docs for RPM/DEB/Docker

grabl committed:
Update dependency to graknlabs/build-tools@7209cb5

grabl committed:
Update dependency to graknlabs/client-java@940daf6

grabl committed:
Update dependency to graknlabs/grakn@625a37f

grabl committed:
Update dependency to graknlabs/client-java@de1e34c

grabl committed:
Update dependency to graknlabs/client-python@eb589e9

grabl committed:
Update dependency to graknlabs/client-nodejs@f6c8fb4

grabl committed:
Update dependency to graknlabs/client-nodejs@d07e53f

Commit:
fix yml structural error (#89)

grabl committed:
Update dependency to graknlabs/build-tools@34730c3

grabl committed:
Update dependency to graknlabs/client-java@d76671e

grabl committed:
Update dependency to graknlabs/grakn@b791c1c

grabl committed:
Update dependency to graknlabs/client-java@8e65aca

grabl committed:
Update dependency to graknlabs/client-python@72c57be

grabl committed:
Update dependency to graknlabs/client-nodejs@05c2ac9

grabl committed:
Update dependency to graknlabs/client-nodejs@ecc3627

Commit:
correct the update example (#92)

grabl committed:
Update dependency to graknlabs/grakn@711fa6e

grabl committed:
Update dependency to graknlabs/client-java@cc1fe5b

grabl committed:
Update dependency to graknlabs/client-nodejs@7b711f6

Commit:
New build-tools dependencies

Commit:
Updated client-python and refactored dependency target names

Commit:
Fixed run-grakn-server command in CI

grabl committed:
//sync-dependencies: graknlabs/client-python@5ab0757

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@d727dde

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@ae10810

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@ae10810

grabl committed:
//sync-dependencies: graknlabs/client-python@cd4738e

Commit:
Added client-nodejs to dependencies.bzl

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@5a2e6b5

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@5a2e6b5

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@5a2e6b5

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@a34260f

Commit:
Add meta keywords and summary for each page (#94)

This PR is for SEO purposes.

The keywords and summaries added to each markdown page are contained within `meta` tags inside the `head` tag of each page. The file that contains content of the `head` tag is a part of the `web-dev` repository (https://github.com/graknlabs/web-dev/blob/master/_includes/head.html).

The `web-dev` counterpart of this PR is: https://github.com/graknlabs/web-dev/pull/37

kasper-piskorski committed:
Refine the schema section (#77)

* refine the schema section

* add behaviours

* add corrections

* corrections

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@f15f981

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@5f96749

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@9f50793

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@d350513

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@44b49d1

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@9f2f043

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@cc70935

Commit:
fix conflict

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@ef71252

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@d3e6234

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@5b4270c

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@c0139ba

Commit:
install-bazel using @graknlabs_build_tools//ci

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@2f9e25e

Commit:
Fixed deploy-docs.py command

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@94ef417

Commit:
Extracted deploy-docs to @graknlabs_build_tools

Commit:
Fixed release-staging and release-production CI jobs

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@b5bc539

Commit:
update reference for client api template (#96)

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@ec2470f

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@3ef0847

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@35a50ea

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@92ddf1f

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@bc26969

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@92ddf1f

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@65e9cc7

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@d454f84

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@2944f72

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@259e56d

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@e69d80b

Commit:
fix broken link

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@e4d87a2

Commit:
rename parab lang to tab (#102)

uses the url param `lang` to `tab` so that we can use it for non-code tabbed content as well.

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@32015cc

Commit:
Bump @graknlabs_build_tools @graknlabs_grakn_core

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@10ab57b

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@d145345

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@b7ae442

Commit:
Updated Grakn and Graql's Maven Group ID

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@ff9e982

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@b8bb880

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@864bb71

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@1bf86f3

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@f46b83f

Commit:
Use rbe host config from build-tools

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@38739d6

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@32d7695

Commit:
use unique values for office registration-number (#100)

sorsaffari reviewed (changes requested):
- screenshots still show queries with the modifiers coming before `get`.
- the slideshow jumps at http://127.0.0.1:4005/docs/workbase/investigation#configure-display-settings
- the limit value changes in http://127.0.0.1:4005/docs/workbase/visualisation#star-queries
- if there is one single screenshot, there is no need to show it as a slideshow

kasper-piskorski committed:
Revise graql overview (#79)

* revise graql overview
* corrections and clarifications

Commit:
Merge branch 'development' of https://github.com/graknlabs/docs into workbase-1.1.2

Commit:
Fix typos (#105)

vmax committed:
Improve installation guide; closes #103 (#106)

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@a1fb403

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@e4472dd

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@83581d4

flyingsilverfin committed:
Add benchmark as transitive dependency (#108)

## What is the goal of this PR?
Fix breaking change caused by the introduction of `benchmark` repository as a Bazel git_repository dependency.

## What are the changes implemented in this PR?
One liner to include `graknlabs_benchmark()` dependency that allows the building of distribution `tar.gz` in CircleCI.

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@3b2e6fa

Commit:
Bump @graknlabs_build_tools and @graknlabs_grakn_core

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@5a92c6f

Commit:
Graql overview revision (#107)

* revise graql overview

* update define link

* revise the extended graql overview conent

* __ to **

* missed out conflict

* corrections and clarifications

* fix extra new line

* fix alignment

* Bump @graknlabs_build_tools @graknlabs_grakn_core

* update the illustration and explainations

* chages in query-structure image and description

Commit:
Set Graql version to 1.0.0

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@1d75f2c

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@4cb5d00

Commit:
Load @graknlabs_benchmark maven deps to enable building grakn from source

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@8d40910

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@1cda459

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@8bedb3e

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@07ae84a

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@fc63822

kasper-piskorski committed:
Fix schema problems filename typo (#109)

Commit:
changes after review

Commit:
Merge branch 'development' of https://github.com/graknlabs/docs into workbase-1.1.2

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@e3f31c9

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@3d2fdc3

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@f03cfec

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@54c6897

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@7915fc9

Commit:
Updated dependencies and new repo addresses

Commit:
Adds GitHub templates (#111)

Commit:
Test links (#112)

## What is the goal of this PR?

make links functional both on our documentation website and this repository.
test the correctness of links.
correct broken links.

**main files to be reviewed are the three in `test/links/` and `.circleci/config.yml`**

## What are the changes implemented in this PR?

**Testing Links**
there are two tests written (and added to CI) to ensure all internal links remain correct at all time.
1. native links: links contained within markdown files and the yaml files used by one or more liquid templates contained within a markdown page. both the target page and the anchor (if provided) are verified.
2. autolinks keywords: these are links defined within the `views/autolink-keywords.js` file. since `autolink-keywords.js` is applied post-render, a mapping of processed links to their corresponding raw version is necessary (this is a limitation that will be avoided once we apply auto link-keywords.js within the process of parsing markdown to html)

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@f698d55

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@92b80bb

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@85b4ad2

Commit:
Fix mistype in phone calls nodejs migration snippet (#113)

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@e1c7e75

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@b442b28

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@9f39863

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@5f9632a

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@ba2f650

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@ddc4b27

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@21d6b15

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@f566679

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@2a1d2ba

Commit:
Update environment variables for release-docs

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@6a626a8

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@35a948b

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@09a4eeb

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@8ee4357

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@d8a9ed0

vmax committed:
Install -all package; import APT key before adding repo (#115)

- Install `-all` packages
- Import GPG key for repo _before_ adding repo, so updates are permitted

Commit:
Update quickstart (#116)

Minor improvements in the content of the Quickstart section

Commit:
Adds docker to install keywords (#117)

sorsaffari commented:
@Irtazaraza resolve the conflicts please

Commit:
Updates configuration content (#118)

sync with 1.5
minor improvements

Commit:
Update clients (#119)

- removes unnecessary lines in front matter
- provides minor linquistic corrections/improvements

Commit:
Updates concept api content (#120)

minor improvements in the overview page of the concept api sectio

Commit:
Update users (#121)

minor improvements in the content for managing users

Commit:
Update query etc (#123)

moves the client references (note panels) to the bottom of the page with a oneliner reference at the top.

Commit:
Update schema (#122)

- post-review of a PR that was already merged (additions to the overview page of the schema section)
- minor linguistic improvements
- removal of unnecessary flags (comments)

Commit:
Resolve rules failing tests (#125)

adds `test-delay` flag to Graql code blocks that contain only patterns.
In the near future, testing Graql patterns will be supported.

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@58b7eb4

grabl committed:
//ci:sync-dependencies: graknlabs/bazel-distribution@53fd544

Commit:
Updated Maven POMs to refer to repo.grakn.ai

grabl committed:
//ci:sync-dependencies: graknlabs/build-tools@9ac89e2

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@732b6e2

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@eeb42d9

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@8c11f88

grabl committed:
//ci:sync-dependencies: graknlabs/grakn@59c303a

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@dc80c1b

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@5accd3a

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@9c44349

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@0c1fe52

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@d7b6e63

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@1c2bc09

grabl committed:
//ci:sync-dependencies: graknlabs/client-java@791141b

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@db549ed

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@3cb3bba

grabl committed:
//ci:sync-dependencies: graknlabs/client-python@833f5f3

Commit:
Bump build-tools, grakn, client-java, client-python

grabl committed:
//ci:sync-dependencies: graknlabs/client-nodejs@cd8f4c1

Commit:
Bump client-nodejs to 1.5.1

Commit:
resolve conflicts

Commit:
remove single picture slide shows

sorsaffari commented:
@Irtazaraza 
`Visualise` >
  -  `Execute Custom Queries`: the sidebar shows `Relation` (instead of `Relations`) on some slides.
  -  `Star Queries`: 6th slide has the concept info panels expanded, none of the other slides in the same slideshow do.

`Schema Designer` >
  -  `Define New Entity Type`, `Define New Attribute Type, `Define New Relation Type` and `Manage Attribute Types`: `Relation Type` keeps moving around as the slideshow is navigated.
  - `Define New Attribute Type`: contains the term `Relationship Type`.

haikalpribadi commented:
Grakn Workbase version will be 1.2.0 @Irtazaraza. make sure to reflect that properly. I'm renaming this PR.

Commit:
rework after review

Commit:
resolve conflicts

Irtazaraza commented:
duplicate of PR #131
docs base changed from development to master
